api/src/main/java/org/openmrs/module/fhir2/providers/r3/RelatedPersonFhirResourceProvider.java

```diff
@@ -63,6 +71,34 @@ public RelatedPerson getRelatedPersonById(@IdParam @Nonnull IdType id) {
 		return RelatedPerson30_40.convertRelatedPerson(relatedPerson);
 	}
 	
+	@Create
+	public MethodOutcome createRelatedPerson(@ResourceParam RelatedPerson relatedPerson) {
+		return FhirProviderUtils.buildCreate(RelatedPerson30_40
+		        .convertRelatedPerson(relatedPersonService.create(RelatedPerson30_40.convertRelatedPerson(relatedPerson))));
+	}
+	
+	@Update
+	public MethodOutcome updateRelatedPerson(@IdParam IdType id, @ResourceParam RelatedPerson relatedPerson) {
+		if (id == null || id.getIdPart() == null) {
+			throw new InvalidRequestException("id must be specified to update");
```

Comment:
@theanandankit please add test for this.

Reply:
@Medhavi-16 thanks, I will add the tests.

Reply:
@Medhavi-16 I added the tests PTAL